### DISCUSSION TITLE
Replacing python manage.py runserver with gunicorn

### DISCUSSION THREAD

**Discussion by anjanesh** [benoitc/gunicorn#3290] [Q&A] [answered]
Our current setup on prod is as follows :
Ubuntu 20.04
Nginx as HTTP server

`sudo service myProject start`

service `myProject` is basically 

```
[Unit]
Description=myProject project
After=network.target

[Service]
User=root
Group=root
WorkingDirectory=/var/www/myProject
Environment="PATH=/var/www/myProject/env/bin/activate"
ExecStart=/bin/bash -c 'source /var/www/myProject/env/bin/activate && python manage.py runserver'
StandardOutput=append:/var/log/myProject/myProject.log
StandardError=append:/var/log/myProject/myProject_error.log

[Install]
WantedBy=multi-user.target
```
Ho do I replace my `myProject` service with **gunicorn** ?

**Comment by aDramaQueen** [comment, marked as answer]
If you install Gunicorn via Pip you get a binary in your venv directory. Therefore: `ExecStart=/var/www/myProject/env/bin/gunicorn`

Everythin else you need, you may find in the [**documentation**](https://docs.gunicorn.org/en/stable/run.html)

> NOTE: You should **not** put your project in the `var` directory. Create a user for your project and put it in the home directory of said user.

**Reply by anjanesh** [reply]
> You should not put your project in the var directory. Create a user for your project and put it in the home directory of said user.

Since this is on a private network and no more ubuntu users, it was setup like this from the beginning. But I wanted to change from runserver to gunicorn.
